src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import DateTime from '../date-time/date-time';
+
+const messages = defineMessages({
+  updated: {
+    id: 'COMPONENT_KIT.FRESHNESS_INDICATOR.UPDATED',
+    defaultMessage: 'Updated',
+    description: 'Label for timestamp value in feed status tooltip',
+  },
+});
+
+function FreshnessIndicatorTooltipTimestamp({ timestamp }) {
+  if (!Number.isFinite(timestamp)) {
+    return (<div />);
+  }
+  return (
+    <div className="freshness-indicator__tooltip__timestamp">
+      <b><FormattedMessage {...messages.updated} />: <DateTime value={new Date(timestamp)} format="numeric" type="time" /></b>
```

Comment:
You should be able to omit `new Date()` from the value and instead pass `value={timestamp}`.